src/cursor/cur_backup.c

```diff
@@ -846,7 +846,9 @@ __backup_stop(WT_SESSION_IMPL *session, WT_CURSOR_BACKUP *cb)
     /* Release all btree names held by the backup. */
     WT_ASSERT(session, !F_ISSET(cb, WT_CURBACKUP_DUP));
     /* If it's not a dup backup cursor, make sure one isn't open. */
-    WT_ASSERT(session, !F_ISSET(session, WT_SESSION_BACKUP_DUP));
+    if (F_ISSET(session, WT_SESSION_BACKUP_DUP)) 
+        WT_RET_MSG(session, EINVAL, "Can't close duplicate cursor since duplicate cursor follows parent cursor");
```

Comment:
I would prefer the message "can't close backup cursor because duplicate backup cursor is still active" @sueloverso what are your thoughts on this?

Reply:
Yes, @jiechenbo your message is the correct one. We're trying to close a backup cursor. I prefer your terms backup cursor and duplicate (i.e. we don't use the term parent cursor for users in backup).

Reply:
In looking at this, I put some thoughts into the ticket. I'm wondering if we really want to do this because this would need to be WT_TRET to allow close to proceed in spite of the error.